build/Microsoft.DotNet.Cli.Monikers.props

```diff
@@ -1,10 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <SdkBrandName>Microsoft .NET Core 1.0.1 - SDK Preview 5</SdkBrandName>
-    <SharedFrameworkBrandName>Microsoft .NET Core 1.0.1 - Runtime</SharedFrameworkBrandName>
-    <SharedHostBrandName>Microsoft .NET Core 1.0.0 - Host</SharedHostBrandName>
-    <HostFxrBrandName>Microsoft .NET Core 1.0.0 - Host FX Resolver</HostFxrBrandName>
+    <SdkBrandName>Microsoft .NET Core 1.0.3 - SDK Preview 5</SdkBrandName>
```

Comment:
At first I thought about calling these properties "LTS" like you suggest, but here's the reason I didn't go with those names:

If/when we decide that the CLI should target the FTS (read 1.1) shared framework, would have to go through all the usages of these variables and potentially switch them around.  Doing it this way, all the normal properties (SharedFrameworkVersion, HostFxrVersion, etc) are the versions that the CLI requires/uses.  The support I added is for an "additional" shared fx to get installed beyond the one that is required.  So if we ever decided to target the FTS, we can just flip the SharedFrameworkVersion to 1.1.0, and the "additional" versions to 1.0.3, and no other change is necessary.

Additionally, when we move to 2.0, and only want to install a single shared fx, we can just remove all the "Additional" properties/usages and be back to normal.  Where if we named them "LTS"/"FTS", it would be weird and potentially incorrect if we then bumped it to "2.1".

Thoughts?